groovy/postInit/metallurgy/StrandCasting.groovy

```diff
@@ -0,0 +1,200 @@
+POLISHING = recipemap('polishing_machine')
+ADVANCED_ARC_FURNACE = recipemap('advanced_arc_furnace')
+FLUID_SOLIDIFIER = recipemap('fluid_solidifier')
+ASSEMBLER = recipemap('assembler')
+// Recipes for the machines and components needed for the multiblocks
+
+// Rolls
+FLUID_SOLIDIFIER.recipeBuilder()
+        .fluidInputs(fluid('vanadium_steel') * 432)
+        .inputs(metaitem('shape.mold.cylinder'))
+        .outputs(metaitem('work_roll.unfinished'))
+        .EUt(30)
+        .duration(80)
+        .build()
+
+POLISHING.recipeBuilder()
+        .inputs(metaitem('work_roll.unfinished'))
+        .fluidInputs(fluid('silicon_dioxide_slurry') * 100)
+        .outputs(item('susy:metallurgy_roll'))
+        .EUt(120)
+        .duration(200)
+        .build()
+
+// Hydraulic Press
+ASSEMBLER.recipeBuilder()
+        .inputs(ore('plateDoubleStainlessSteel'))
+        .inputs(ore('stickStainlessSteel'))
+        .inputs(ore('boltStainlessSteel') * 4)
+        .inputs(ore('ringRubber') * 2)
+        .inputs(ore('pipeNormalFluidStainlessSteel'))
+        .fluidInputs(fluid('silicone_oil') * 100)
+        .outputs(item('susy:metallurgy'))
+        .EUt(120)
+        .duration(300)
+        .build()
+
+// Flying Shear Saw
+ASSEMBLER.recipeBuilder()
+        .inputs(metaitem('component.grinder.diamond'))
+        .inputs(ore('stickStainlessSteel'))
+        .inputs(item('gregtech:metal_casing', 4))
+        .inputs(metaitem('electric.motor.hv'))
+        .inputs(metaitem('cableGtSingleGold') * 2)
+        .outputs(item('susy:metallurgy_2'))
+        .EUt(16)
+        .duration(100)
+        .build()
+
+// Monel 500 casings
+ASSEMBLER.recipeBuilder()
+        .inputs(metaitem('plateMonel500') * 6)
+        .outputs(metaitem('frameHsla980X'))
+        .outputs(item('susy:susy_multiblock_casing', 5) * 4)
+        .EUt(16)
+        .duration(100)
+        .build()
+
+// Pipe casing
+crafting.addShaped("susy:monel_500_pipe_casing", item('susy:susy_multiblock_casing', 6) * 2, [
+    [ore('plateMonel500'), ore('pipeNormalFluidMonel500'), ore('plateMonel500')],
+    [ore('pipeNormalFluidMonel500'), metaitem('frameHsla980X'), ore('pipeNormalFluidMonel500')],
+    [ore('plateMonel500'), ore('pipeNormalFluidMonel500'), ore('plateMonel500')]
+])
+
+// Machines
+// Gas atomizer
```

Comment:
wasn't this machine meant to produce plasma for lithography

Reply:
No, not really. Here's an informative diagram:
![image](https://github.com/user-attachments/assets/514779a8-75ec-444b-bcd7-59bfbbb67957)
It's mostly about making homogeneous dusts.